.github/dependabot.yml

```diff
@@ -0,0 +1,25 @@
+version: 2
+updates:
+  - package-ecosystem: "nuget"
+    directory: "/src/MockClassifier.Api/"
+    schedule:
+      interval: "daily"
+      time: "22:00"
```

Comment:
What do you think about adding the timezone here?   Same discussion as in the previous PR: https://github.com/buerokratt/DMR/pull/64#discussion_r929583036

Reply:
I'm resolving this convo in light of your response on the above thread. Thanks!